.editorconfig

```diff
@@ -1,7 +0,0 @@
-[*]
```

Comment:
How does `.editorconfig` work? Do we have to install any [VSCode plugin](https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig)? The [documentation](https://editorconfig.org/#:~:text=To%20use%20EditorConfig%20with%20one%20of%20these%20editors%2C%20you%20will%20need%20to%20install%20a%20plugin.) mentions this, but `.vscode/extensions.json` doesn't include it, so not sure.

Reply:
editorconfig has been removed. previously, we were using it through prettier, not directly. see docs: https://prettier.io/docs/en/options.html#print-width